src/app/WriteHandler.cpp

```diff
@@ -345,15 +418,44 @@ CHIP_ERROR WriteHandler::ProcessGroupAttributeDataIBs(TLV::TLVReader & aAttribut
         iterator = groupDataProvider->IterateEndpoints(fabric);
         VerifyOrExit(iterator != nullptr, err = CHIP_ERROR_NO_MEMORY);
 
+        bool shouldReportListWriteEnd = ShouldReportListWriteEnd(mProcessingAttributePath, mLastAttributeIsList, dataAttributePath);
+        bool shouldReportListWriteBegin = ShouldReportListWriteBegin(mProcessingAttributePath, dataAttributePath);
+        auto lastAttributePath          = mProcessingAttributePath.ValueOr(ConcreteAttributePath());
+
+        mLastAttributeIsList = dataAttributePath.IsListOperation();
+        mProcessingAttributePath.SetValue(dataAttributePath);
+
         while (iterator->Next(mapping))
```

Comment:
Can I have IBs in a group message that reference specific endpoints? If not, should we permit that?

@bzbarsky-apple

Reply:
I believe right now we cannot, and the spec disallows it, no?